keras/layers/recurrent.py

```diff
@@ -365,6 +376,8 @@ def step(self, x, states):
             h = K.dot(x * B_W, self.W) + self.b
 
         output = self.activation(h + K.dot(prev_output * B_U, self.U))
+        if 0 < self.zoneout_h < 1:
```

Comment:
Is 1 a valid value? Could be useful if you want to examine an RNN with no memory just for comparison purposes.

Instead of silently doing nothing if zoneout is >= 1, should raise a ValueError.

This condition should match the uses_learning_phase = True statement above.